labrad/oauth.py

```diff
@@ -90,19 +94,20 @@ def get_token(client_id, client_secret, headless=False, timeout=60):
     """
     cached = _get_cached_token(client_id)
     if cached is None:
-        print('No cached token. Logging in...')
+        log.debug('No cached token. Logging in. client_id=%s', client_id)
```

Comment:
Passing the args to the log method separately means we don't do the string interpolation work unless there's actually a handler registered that can accept the log message. In this case, the work saved is trivial, but I think it's a good habit to get in with logging.